core/eolearn/tests/test_eodata.py

```diff
@@ -112,10 +105,26 @@ def test_invalid_characters():
         eop.data_timeless['mask.npy'] = np.arange(3 * 3 * 2).reshape(3, 3, 2)
 
 
+def test_repr(test_eopatch_path):
+    test_eopatch = EOPatch.load(test_eopatch_path)
+
+    for eopatch in [test_eopatch, EOPatch()]:
+        repr_str = repr(eopatch)
+
+        assert isinstance(repr_str, str)
+        assert repr_str == eopatch.__repr__()
```

Comment:
is this not the same thing?

Reply:
Hm, I though that `repr` function falls back to `__str__` if `__repr__` is not defined but now I see that is actually the default definition of `__repr__`. Will fix.